README.md

```diff
@@ -1,2 +1,2 @@
 # bcvm
-LLVM bitcode interpreter.
+LLVM bitcode interpreter written in Rust.
```

Comment:
I wouldn't usually do this for a change like this, but to show you the ropes I'll make a suggestion for a change now:

Can we please change this to `An LLVM bitcode...`?